src/controllers/softtakeover.cpp

```diff
@@ -60,14 +66,21 @@ bool SoftTakeoverCtrl::ignore(ControlObject* control, double newParameter) {
     return ignore;
 }
 
+void SoftTakeoverCtrl::ignoreNext(ControlObject* control) {
```

Comment:
Is it possible the CO could be something other than NULL (that evaluates to FALSE) in light of the isNaN compiler-assumption stuff? If (true) handles such cases.